{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -7,57 +7,47 @@ Supports registering, viewing, and updating user accounts.
 
 `POST` `users/`
 
+Parameters:
+
+Name     | Type   | Description
+---------|--------|---
+username | string | The username for the new user.
+password | string | The password for the new user (plaintext).
+
 *Note:*
 
 - *Not* **Authorization Protected**
 
 
 **Response**:
+If a User with given `username` doesn't exist, you'll get a 201:
```

Comment:
Since validation errors are standardized through DRF, it would be repetitive to repeat the `400 Bad Request Response` case in every end point.  I'd prefer to leave this information out so that developers can easily get the information they need, without looking through a wall of text. If you check out the GitHub [api documentation](https://developer.github.com/v3/issues/#list-issues) they've chosen to do this as well.